docs/quickstart.rst

```diff
@@ -27,22 +33,34 @@ Change ``/hello`` and ``/world`` to some actual paths on the website/service you
     [2021-07-24 09:58:46,215] .../INFO/locust.main: Starting web interface at http://0.0.0.0:8089
     [2021-07-24 09:58:46,285] .../INFO/locust.main: Starting Locust |version|
 
+Congratulations, you've run a Locust test!
+
+.. TODO you can't run a test first, and open the web interface to connect to that right? You either need to run it from the web interface, or specify the details and run in headless
+
+Open the :ref:`web interface <interface>` to view the test results, or restart locust with the ``--headless`` parameter to view them on the :ref:`command line <headless>`.
+
+.. _interface:
+
 Locust's web interface
 ======================
 
-Open http://localhost:8089
+Locust's web interface is available by default on http://localhost:8089 . Specify some basic test parameters and a host to test, and once it's started, get reports on the running tests.
 
 .. image:: images/webui-splash-screenshot.png
-
-| Provide the host name of your server and try it out!
+    :alt: Screenshot of the starting page of the Locust web interface
 
 The following screenshots show what it might look like when running this test using 50 concurrent users, with a ramp up rate of 1 user/s
 
 .. image:: images/webui-running-statistics.png
+    :alt: Screenshot of the Locust web interface statistics page
 
 | Under the *Charts* tab you'll find things like requests per second (RPS), response times and number of running users:
 
 .. image:: images/bottlenecked_server.png
+    :alt: Screenshot of the Locust web interface charts page
+
+
+.. TODO can we at least point folks to external references on interpreting these graphs?
```

Comment:
I'm writing some blog articles (https://medium.com/locust-cloud/load-testing-part-1-0f2332aa4a07, https://medium.com/locust-cloud/smoke-stress-spike-soak-and-recovery-5-essential-load-test-profiles-4606168d9c1a), we might want to link them somewhere :) 

I might write an entire post about interpreting results... but maybe leave this for later.